maplibre/src/lib.rs

```diff
@@ -183,6 +224,12 @@ where
         self
     }
 
+    /// Builds the UninitializedMap with the given configuration.
+    ///
+    /// # Panics
+    ///
+    /// * Panics if no schedule method has been configured.
+    /// * Panics if no http client has been configured.
```

Comment:
I think we can leave this undocumented right now. We will remove all unwraps in a future version.